metricbeat/module/system/services/_meta/fields.yml

```diff
@@ -0,0 +1,62 @@
+- name: services
+  type: group
+  release: beta
+  description: >
+    metrics for system services
+  fields:
+    - name: name
+      type: keyword
+      description: The name of the systemd service
```

Comment:
Thinking on systemd, we should use the same [common fields](https://www.elastic.co/guide/en/beats/journalbeat/current/exported-fields-common.html#_systemd_2) as journalbeat, for corelation of events. For example for the unit name journalbeat uses `systemd.unit`.

As this module will be generic to other service management modules we may think in having sets of fields specific to the platform, in that case for this example we would have `metricbeat.services.name` as generic name (that could be without the `.service` part if it makes sense), and `systemd.unit`.

If we go towards having generic fields we shouldn't use systemd-specific words as `systemd`, or `unit` in the descriptions.

Reply:
Actually for the generic fields we could make use of ECS fields https://www.elastic.co/guide/en/ecs/1.2/ecs-service.html

Reply:
Alright, I tried to make the docs a tad more generic, and it _should_ be ECE compliant now.